packages/tabs/src/Tabs.ts

```diff
@@ -229,15 +233,40 @@ export class Tabs extends SizedMixin(Focusable, { noDefaultSize: true }) {
         return {};
     }
 
-    protected override manageAutoFocus(): void {
+    override async getUpdateComplete(): Promise<boolean> {
+        const complete = await super.getUpdateComplete();
+
         const tabs = [...this.children] as Tab[];
         const tabUpdateCompletes = tabs.map((tab) => {
             if (typeof tab.updateComplete !== 'undefined') {
                 return tab.updateComplete;
             }
             return Promise.resolve(true);
         });
-        Promise.all(tabUpdateCompletes).then(() => super.manageAutoFocus());
+
+        await Promise.all(tabUpdateCompletes);
+        return complete;
+    }
+
+    public scrollToSelection(): void {
+        if (this.enableTabsScroll && this.selected) {
+            this.updateComplete.then(() => {
+                const selectedTab = this.querySelector(
+                    `[role="tab"][value="${this.selected}"]`
+                ) as Tab;
+                selectedTab?.scrollIntoView();
+            });
+        }
+    }
+
+    protected override updated(
+        changedProperties: PropertyValueMap<this>
+    ): void {
+        super.updated(changedProperties);
+
+        if (changedProperties.has('selected') && this.shouldAutoscroll) {
```

Comment:
If we don't remove `this.shouldAutoscroll`, would it be better as a gate internal to `scrollToSelection()`?